2022-04-11 14:15:27,955 INFO    MainThread:53391 [wandb_setup.py:_flush():71] setting env: {}
2022-04-11 14:15:27,956 INFO    MainThread:53391 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-11 14:15:27,956 INFO    MainThread:53391 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-11 14:15:27,956 INFO    MainThread:53391 [wandb_init.py:_log_setup():371] Logging user logs to /share/mastoc/projects/erasmus/deepchange/codebase/DeepChange/ltae/wandb/run-20220411_141527-2bn51nvx/logs/debug.log
2022-04-11 14:15:27,956 INFO    MainThread:53391 [wandb_init.py:_log_setup():372] Logging internal logs to /share/mastoc/projects/erasmus/deepchange/codebase/DeepChange/ltae/wandb/run-20220411_141527-2bn51nvx/logs/debug-internal.log
2022-04-11 14:15:27,957 INFO    MainThread:53391 [wandb_init.py:init():404] calling init triggers
2022-04-11 14:15:27,957 INFO    MainThread:53391 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-04-11 14:15:27,957 INFO    MainThread:53391 [wandb_init.py:init():460] starting backend
2022-04-11 14:15:27,957 INFO    MainThread:53391 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-11 14:15:27,980 INFO    MainThread:53391 [backend.py:ensure_launched():216] starting backend process...
2022-04-11 14:15:27,990 INFO    MainThread:53391 [backend.py:ensure_launched():222] started backend process with pid: 53413
2022-04-11 14:15:27,992 INFO    MainThread:53391 [wandb_init.py:init():469] backend started and connected
2022-04-11 14:15:28,001 INFO    MainThread:53391 [wandb_init.py:init():533] updated telemetry
2022-04-11 14:15:28,024 INFO    MainThread:53391 [wandb_init.py:init():563] communicating current version
2022-04-11 14:15:29,567 INFO    MainThread:53391 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-11 14:15:29,568 INFO    MainThread:53391 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-11 14:15:30,000 INFO    MainThread:53391 [wandb_init.py:init():606] starting run threads in backend
2022-04-11 14:15:35,024 INFO    MainThread:53391 [wandb_run.py:_console_start():1810] atexit reg
2022-04-11 14:15:35,026 INFO    MainThread:53391 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-11 14:15:35,027 INFO    MainThread:53391 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-11 14:15:35,031 INFO    MainThread:53391 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-11 14:15:35,031 INFO    MainThread:53391 [wandb_init.py:init():633] run started, returning control to user process
2022-04-11 14:16:05,355 INFO    MainThread:53391 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-11 14:16:05,357 INFO    MainThread:53391 [wandb_run.py:_restore():1752] restore
2022-04-11 14:16:07,720 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 780
  total_bytes: 780
}

2022-04-11 14:16:08,168 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 780
  total_bytes: 780
}

2022-04-11 14:16:09,099 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 780
  total_bytes: 15696
}

2022-04-11 14:16:09,201 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 780
  total_bytes: 16511
}

2022-04-11 14:16:09,304 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 780
  total_bytes: 16511
}

2022-04-11 14:16:09,406 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6023
  total_bytes: 16511
}

2022-04-11 14:16:09,508 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15669
  total_bytes: 16511
}

2022-04-11 14:16:09,611 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15696
  total_bytes: 16511
}

2022-04-11 14:16:09,715 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:09,818 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:09,920 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:10,023 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:10,125 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:10,380 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}

2022-04-11 14:16:10,948 INFO    MainThread:53391 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16511
  total_bytes: 16511
}
local_info {
}

2022-04-11 14:16:12,462 INFO    MainThread:53391 [wandb_run.py:_append_files():2180] logging synced files
